docs-2.0/20.appendix/0.FAQ.md

```diff
@@ -424,15 +424,15 @@ The notification time before the license expires is different for the full versi
 
 - For the full version license:
 
-  - Nebula Graph Enterprise Edition: 7 days before the license expires, there is an expiration reminder when Nebula Graph is started, and a 3-day buffer period after expiration. During the buffer period, you can continue using Nebula Graph. After the buffer period ends, Nebula Graph will be down and cannot be started.
+  - Within 30 days before the license expires or on the day the license expires, there is an expiration reminder when Nebula Graph/Dashboard/Explorer is started.
```

Comment:
After the license expires, can users receive such a reminder if it is still that day? Users should not be able to log into the database to see the reminder, right?

Reply:
> After the license expires, can users receive such a reminder if it is still that day? Users should not be able to log into the database to see the reminder, right?

;) users can log into the database or service on the day the license expires and will receive a license expiration reminder.